orchestrator-salt/src/main/resources/salt/salt/postgresql/root-certs.sls

```diff
@@ -10,7 +10,7 @@ create-root-certs-file:
     - group: root
     - mode: 644
 
-{% if postgresql.ssl_restart_required == True %}
+{% if postgresql.ssl_restart_required == True and postgresql.ssl_restart_required == True and "manager_server" in grains.get('roles', []) %}
```

Comment:
Duplicated condition; I suppose you intended using `postgresql.ssl_enabled == True` instead. :smile: